cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -109,6 +109,19 @@
     <div class="sidebar-content">
         <p><spring:message code="screen.welcome.security" /></p>
 
+        <c:if test="${!empty pac4jUrls}">
+            <div id="list-providers">
+                <h3><spring:message code="screen.welcome.label.loginwith" /></h3>
+                <form>
+                    <ul>
```

Comment:
No problem. 

I was thinking to modify the demo server to include this configuration by default, and then thought images would look good there for the demo but that's a customization we can make to the demo server. No big deal.